docs/dotnet-api-builder.md

```diff
@@ -6,16 +6,35 @@ For the following examples, we are using our own [W9s custom API](https://develo
 ## Quick Start
 
 ```csharp
+using Mindee;
+using Mindee.Input;
+using Mindee.Http;
+using Mindee.Parsing;
+
+string apiKey = "my-api-key";
+string filePath = "/path/to/the/file.ext";
+
+MindeeClient mindeeClient = new MindeeClient(apiKey);
+
+// load an input source
+var inputSource = new LocalInputSource(filePath);
+
+// Set the endpoint configuration
 CustomEndpoint myEndpoint = new CustomEndpoint(
     endpointName: "my-endpoint",
-    accountName: "my-account",
+    accountName: "my-account"
     // optionally, lock the version
-    //version: "1.1"
+    //, version: "1.1"
```

Comment:
Was leave a trailing comma not an option? This looks unintuitive

Reply:
I agree it's not very nice but trailing commas won't compile